lpc55_sign/src/signed_image.rs

```diff
@@ -78,7 +78,7 @@ pub fn sign_chain(
         .cert_paths
         .iter()
         .flat_map(|path| {
-            let cert_bytes = std::fs::read(prefix.join(&path)).unwrap();
+            let cert_bytes = std::fs::read(prefix.join(path)).unwrap();
```

Comment:
(this would also be a very polite place for `context` if somewhat less likely)

Reply:
Fixed, although we're still `unwrap`'ing it because of the loop structure (alas)